doc-surrealdb_versioned_docs/version-latest/integration/rpc.mdx

```diff
@@ -790,6 +845,286 @@ query [ sql, vars ]
 }
 ```
 
+## `run`
+
+This method allows you to execute built-in functions, custom functions, or machine learning models with optional arguments.
+
+```json title="Method Syntax"
+run [ func_name, version?, args? ]
+```
+
+### Parameters
+
+<table>
+    <thead>
+        <tr>
+            <th colspan="2" scope="col">Parameter</th>
+            <th colspan="2" scope="col">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td colspan="2" scope="row" data-label="Parameter">
+                <code>func_name</code>
+                <l className='yellow'>REQUIRED</l>
+            </td>
+            <td colspan="2" scope="row" data-label="Description">
+                The name of the function or model to execute. Prefix with `fn::` for custom functions or `ml::` for machine learning models.
+            </td>
+        </tr>
+        <tr>
+            <td colspan="2" scope="row" data-label="Parameter">
+                <code>version</code>
+                <l className='grey'>OPTIONAL</l>
+            </td>
+            <td colspan="2" scope="row" data-label="Description">
+                The version of the function or model to execute.
+            </td>
+        </tr>
+        <tr>
+            <td colspan="2" scope="row" data-label="Parameter">
+                <code>args</code>
+                <l className='grey'>OPTIONAL</l>
+            </td>
+            <td colspan="2" scope="row" data-label="Description">
+                The arguments to pass to the function or model.
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+
+### Executing a built-in function
+
+```json title="Request"
+{
+    "id": 1,
+    "method": "run",
+    "params": [ "time::now" ]
+}
+```
+
+```json title="Response"
+{
+    "id": 1,
+    "result": "2024-09-15T12:34:56Z"
+}
+```
+
+### Executing a custom function
+
+```json title="Request"
+{
+    "id": 1,
+    "method": "run",
+    "params": [ "fn::calculate_discount", [ 100, 15 ] ]
```

Comment:
[100, 15] will be interpreted as the version, it should be [ "fn::calculate_discount", null, [ 100, 15 ] ]